### PR TITLE
100% coverage of `net.minecraft.data`

### PR DESCRIPTION
**Pull request by eristhea**
- coverage. mostly mojanged instead of yarned, unless it goes against existing names
- bumps plugin to 1.4 (should make a devlog about this!)
- all refactors are for mappings that are fully wrong at this point
- uses names from #496 as a base
- hardens spellcheck against errors when a package name has duplicate words

### CODE REVIEW

File: buildSrc/src/main/resources/minecraft_specific_words.txt [hunk 1]
```diff
@@ -6,6 +6,7 @@ gameplay
 griefing
 minceraft
 minecraft
+datapack
```
Comment (supersaiyansubtlety) [inline buildSrc/src/main/resources/minecraft_specific_words.txt]: Wasn't this removed in favor of always using "data pack"?

Comment (eristhea) [reply buildSrc/src/main/resources/minecraft_specific_words.txt]: you're right, I'll fix it

File: mappings/net/minecraft/data/server/advancement/HusbandryAdvancementTabGenerator.mapping [hunk 2]
```diff
@@ -7,15 +7,25 @@ CLASS net/minecraft/unmapped/C_vfenxpjt net/minecraft/data/server/advancement/Hu
...
 		ARG 1 task
 	METHOD m_mjrojbaf appendBucketAllFishCriterion (Lnet/minecraft/unmapped/C_kdwyuhdb$C_dukbjvrv;)Lnet/minecraft/unmapped/C_kdwyuhdb$C_dukbjvrv;
+		ARG 0 task
 		ARG 1 task
 	METHOD m_msnwtfou appendLeashAllFrogTypesCriterion (Lnet/minecraft/unmapped/C_kdwyuhdb$C_dukbjvrv;)Lnet/minecraft/unmapped/C_kdwyuhdb$C_dukbjvrv;
+		ARG 0 task
 		ARG 1 task
 	METHOD m_ropwgspa appendCatchAllFishCriterion (Lnet/minecraft/unmapped/C_kdwyuhdb$C_dukbjvrv;)Lnet/minecraft/unmapped/C_kdwyuhdb$C_dukbjvrv;
+		ARG 0 task
 		ARG 1 task
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/data/server/advancement/HusbandryAdvancementTabGenerator.mapping]: these appear to all have two `task` args

Comment (eristhea) [reply mappings/net/minecraft/data/server/advancement/HusbandryAdvancementTabGenerator.mapping]: in enigma all that's visible is the first one, I'll remove the arg 1 from all of them

File: mappings/net/minecraft/data/server/tag/TagsProvider.mapping [hunk 3]
```diff
@@ -11,4 +15,6 @@ CLASS net/minecraft/unmapped/C_hzdskvsx net/minecraft/data/server/tag/TagsProvid
 			ARG 1 builder
 			ARG 2 valueToKey
 		METHOD m_agatuolp add ([Ljava/lang/Object;)Lnet/minecraft/unmapped/C_hzdskvsx$C_wosjfhof;
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/data/server/tag/TagsProvider.mapping]:
```suggestion
		METHOD m_agatuolp addAll ([Ljava/lang/Object;)Lnet/minecraft/unmapped/C_hzdskvsx$C_wosjfhof;
```
to match `Collection.addAll`?

File: mappings/net/minecraft/recipe/RecipeCodecThingTODO.mapping [hunk 4]
```diff
@@ -0,0 +1 @@
+CLASS net/minecraft/unmapped/C_fvoqrgef net/minecraft/recipe/RecipeCodecThingTODO
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/recipe/RecipeCodecThingTODO.mapping]: was this left in by mistake?

**Review by supersaiyansubtlety — COMMENTED** [review]
Looks good, just a couple questions

File: mappings/net/minecraft/recipe/RecipeCodecThingTODO.mapping (hunk 4 above)
Comment (eristhea) [reply mappings/net/minecraft/recipe/RecipeCodecThingTODO.mapping]: 100%

**Review by supersaiyansubtlety — APPROVED** [review]

**Review by OroArmor — APPROVED** [review]

**Pull request merged**